entries/e/epo.org.json

```diff
@@ -0,0 +1,16 @@
+{
+  "European Patent Office": {
+    "domain": "epo.org",
+    "tfa": [
+      "sms",
+      "email",
+      "totp"
+    ],
+    "documentation": "https://www.epo.org/en/applying/myepo-services/get-access",
+    "notes": "2FA for all MyEPO Services",
```

Comment:
This note isn't necessary, as the normal assumption would be that it is enabled throughout the website.
```suggestion
```